.gitignore

```diff
@@ -258,4 +258,7 @@ paket-files/
 
 # Python Tools for Visual Studio (PTVS)
 __pycache__/
-*.pyc
\ No newline at end of file
+*.pyc
+
+
+wimm\.Guardian/GlobalSuppressions\.cs
```

Comment:
This file doesn't seem to exist. No real hard in it being here if it's just something your tooling added... just want to confirm it's not indicative of something that was supposed to be included.

Reply:
I use RoslynatorCodeSomethingOrOther and since TWAE is on things that it says are "Warnings" can mess up my building. Shouldn't be included sorta like a user file really

Reply:
D'oh, of course it doesn't exist this is a .gitignore. Lost the plot, my mistake.